packages/rum-react/test/specs/get-with-transaction.spec.js

```diff
@@ -91,4 +93,14 @@ describe('withTransaction', function() {
       'test-name is not instrumented since component property is not provided'
     )
   })
+
+  it('should not instrument the route when rum is inactive', () => {
+    const transactionService = serviceFactory.getService('TransactionService')
+    spyOn(transactionService, 'startTransaction')
+
+    configService.setConfig({ active: false })
+    TestComponent(new ApmBase(serviceFactory, true))
+
+    expect(transactionService.startTransaction).not.toHaveBeenCalled()
```

Comment:
Maybe instead of this we should check if the Component is the same as the WrappedComponent!

Reply:
Sure, We can have both of those checks.